Fixes upstream's handling of default plugins. gconf expects a comma-separated
list, not a space-separated list.

This patch by Thierry Reding.

--- a/plugins/compiz.schemas.in
+++ b/plugins/compiz.schemas.in
@@ -8,7 +8,7 @@
             <owner>compiz</owner>
             <type>list</type>
             <list_type>string</list_type>
-            <default>[gconf @DEFAULT_PLUGINS@]</default>
+            <default>[@DEFAULT_PLUGINS@]</default>
             <locale name="C">
                 <short>Active Plugins</short>
                 <long>List of currently active plugins</long>
